lighthouse/lighthouse-auth.mjs

```diff
@@ -57,22 +57,30 @@ async function login(page, origin) {
 export async function loadInitialPage(page, origin, timeout = 60_000) {
   let originResponse = await page.goto(origin);
 
-  if (originResponse.status && originResponse.status !== 404) {
+  if (pageFound(originResponse)) {
     return Promise.resolve(originResponse);
   }
 
   return await new Promise((resolve) => {
     const initialLoadId = setInterval(async () => {
-      if (originResponse.status && originResponse.status !== 404) {
+      if (pageFound(originResponse)) {
         resolve(originResponse);
         clearInterval(initialLoadId);
       } else {
```

Comment:
This branch isn't needed anymore, right?

Reply:
It is needed because otherwise it never re-queries the page.